infoschema/builder.go

```diff
@@ -270,6 +268,22 @@ func (b *Builder) applyCreateTable(m *meta.Meta, dbInfo *model.DBInfo, tableID i
 			fmt.Sprintf("(Table ID %d)", tableID),
 		)
 	}
+
+	err = b.applyPlacementUpdate(placement.GroupID(tableID))
+	if err != nil {
+		return nil, err
+	}
```

Comment:
For now, table has no placement rules.

Reply:
I thought `truncatePartition` schemaDiff is handled here... Try later.

Reply:
Yes, you can handle it here. But `groupID == tableID` doesn't exist, because there's no rule for tables.

Reply:
It is removed.